jaxley/modules/base.py

```diff
@@ -2422,7 +2426,8 @@ def __init__(
         )
 
         self.channels = self._channels_in_view(pointer)
-        self.membrane_current_names = [c._name for c in self.channels]
+        self.membrane_current_names = [c.current_name for c in self.channels]
```

Comment:
Think this was a bug, it wasn't actually possible to record channels currents from a `View` because of the naming